http/mixed-content.json

```diff
@@ -0,0 +1,322 @@
+{
+  "http": {
+    "mixed_content": {
+      "__compat": {
+        "description": "Blocks some or all insecure mixed content",
+        "mdn_url": "https://developer.mozilla.org/docs/Web/Security/Mixed_content",
+        "spec_url": "https://www.w3.org/TR/mixed-content/#intro",
+        "support": {
+          "chrome": {
+            "version_added": "79"
+          },
+          "chrome_android": "mirror",
+          "edge": "mirror",
+          "firefox": {
+            "version_added": "23"
+          },
+          "firefox_android": "mirror",
+          "ie": {
+            "version_added": false
+          },
+          "oculus": "mirror",
+          "opera": "mirror",
+          "opera_android": "mirror",
+          "safari": {
+            "version_added": true
+          },
+          "safari_ios": "mirror",
+          "samsunginternet_android": "mirror",
+          "webview_android": "mirror"
+        },
+        "status": {
+          "experimental": false,
+          "standard_track": true,
+          "deprecated": false
+        }
+      },
+      "allow_file_urls": {
+        "__compat": {
+          "description": "Allow mixed content from <code>file:</code> URLs",
+          "spec_url": "https://www.w3.org/TR/mixed-content/#intro",
+          "support": {
+            "chrome": {
+              "version_added": true
+            },
+            "chrome_android": "mirror",
+            "edge": "mirror",
+            "firefox": {
+              "version_added": true
+            },
+            "firefox_android": "mirror",
+            "ie": {
+              "version_added": false
+            },
+            "oculus": "mirror",
+            "opera": "mirror",
+            "opera_android": "mirror",
+            "safari": {
+              "version_added": false
+            },
+            "safari_ios": "mirror",
+            "samsunginternet_android": "mirror",
+            "webview_android": "mirror"
+          },
+          "status": {
+            "experimental": false,
+            "standard_track": true,
+            "deprecated": false
+          }
+        }
+      },
+      "allow_localhost_url": {
+        "__compat": {
+          "description": "Allow mixed content from localhost addresses (<code>http://localhost/</code> and <code>http://*.localhost/</code>).",
+          "spec_url": "https://www.w3.org/TR/mixed-content/#intro",
+          "support": {
+            "chrome": {
+              "version_added": true
```

Comment:
I know this to be true, but not the version. What should I use?

Reply:
```suggestion
              "version_added": "≤79"
```